plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/extractor/PluginMetadata.java

```diff
@@ -0,0 +1,17 @@
+package io.jenkins.tools.pluginmodernizer.core.extractor;
+
+import java.util.List;
+import java.util.Map;
+
+import org.openrewrite.maven.tree.*;
+
+public class PluginMetadata {
```

Comment:
I would keep it very very simple for now.

Probably a first iteration would be

pluginName
parentVersion
jenkinsVersion

What do you think ?

The rest could be added later if we need to

Reply:
```    
    private String pluginName;
    private boolean isLicensed;
    private boolean hasDevelopersTag;
    private boolean hasJavaLevel;
    private boolean usesHttps;
    private List<Dependency> dependencies;
    private String jenkinsVersion;
    private Parent pluginParent;
    private Map<String, String> properties;
```

Already extracted the things in this list. Will commit after resolving spotbugs and checkstyle issues.